docs/Server.md

```diff
@@ -345,7 +345,7 @@ fastify.get('/', (request, reply) => {
   console.log(request.raw.ips)
   console.log(request.raw.hostname)
   request.raw.log('Hello')
-  reply.res.log('World')
+  reply.raw.log('World')
```

Comment:
This is the perfect example of our current inconsistency :)

Reply:
That should just be `reply.log` anyway.

Reply:
This is also true :)